packages/block-editor/src/components/block-tools/index.js

```diff
@@ -251,6 +245,11 @@ export default function BlockTools( {
 					name="__unstable-block-tools-after"
 					ref={ blockToolbarAfterRef }
 				/>
+				{ ! isTyping && (
+					<InsertionPoint
+						__unstableContentRef={ __unstableContentRef }
+					/>
+				) }
```

Comment:
Note that the order previously didn't matter here, it was available only to pointer devices.
Also note that the zoom-out inserters are actually right below this one now.